OMPython/__init__.py

```diff
@@ -238,8 +239,11 @@ def sendExpression(self, command):
             self._omc = None
             return result
           else:
-            answer = OMTypedParser.parseString(result)
-            return answer
+            try:
+              answer = OMTypedParser.parseString(result)
+              return answer
+            except:
```

Comment:
what about adding a parsed FLag in send Expression , so that when i want only unparsed result i can specify the flag to be false , like in this commit https://github.com/arun3688/OMPython/commit/ee3287a4b7da62c3cf010e9dd41ec9bab424e5c2